server/api/image_uploader.py

```diff
@@ -0,0 +1,10 @@
+import cloudinary
+import cloudinary.uploader
+import cloudinary.api
+
+
+def imageUploader(image_url,image_preset,cloud_name):
+    return cloudinary.uploader.unsigned_upload(image_url,image_preset,cloud_name=cloud_name)['secure_url']
+
+
+
```

Comment:
also unneeded extra lines.
Do you need all 3 imports or just cloudinary.uploader?
Also, snake_case for function names in Python and spaces between the parameters please :)